src/component/@common/Dropdown/DropdownItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentProps } from 'react';
+
+import { useDropdownContext } from './context/dropdownContext';
+import { dropdownItemStyle } from './Dropdown.styles';
+
+interface DropdownItemProps extends ComponentProps<'li'> {
+  handleClick: (content: string) => void;
+  content: string;
+}
+
+const DropdownItem = ({ content, handleClick, ...rest }: DropdownItemProps) => {
+  const { close } = useDropdownContext();
+
+  const handleSelect = () => {
+    close();
+    handleClick?.(content);
+  };
+
+  return (
+    <li onClick={handleSelect} css={dropdownItemStyle} {...rest}>
+      {content}
+    </li>
+  );
+};
+
+export default DropdownItem;
```

Comment:
이렇게 value의 값을 제네릭으로 받아와서 처리해주면 Dropwdown을 사용하는 쪽에서 타입을 다루기가 쉽답니다!

```suggestion
interface DropdownItemProps<T extends string = string> extends Omit<ComponentProps<'li'>, 'onClick'> {
  content: T;
  onClick: (content: T) => void;
}

function DropdownItem<T extends string = string>({ content, onClick, ...rest }: DropdownItemProps<T>) {
  const { close } = useDropdownContext();

  const handleSelect = () => {
    close();
    onClick?.(content);
  };

  return (
    <li onClick={handleSelect} css={dropdownItemStyle} {...rest}>
      {content}
    </li>
  );
}

export default DropdownItem;
```